jekyll/_cci2/deployment-examples.md

```diff
@@ -0,0 +1,685 @@
+---
+layout: classic-docs
+title: "Deployment Examples"
+short-title: "Deployment Examples"
+---
+This document presents example config for a variately of popular deployment targets. Many of the examples use orbs: CircleCI and partners have developed a catalogue of orbs that enable you to quickly deploy applications with minimal config. Details of all orbs can be found in the [CircleCI Orbs Registry](https://circleci.com/orbs/registry/).
+
+* TOC
+{:toc}
+
+#### Notes on Examples
+{:.no_toc}
+
+* In order to use orbs you must use `version 2.1` config, and enable pipelines for your project. 
+* We have indicated where you need to specify a [docker image for your job]({{ site.baseurl }}/2.0/optimizations/#docker-image-choice) with `<docker-image-name-tag>`.
+* If you wish to remain using `version 2.0` config, or are using a self-hosted installation of CircleCI Server, the examples shown here are still relevant because you can view the expanded orb source within the [Orbs Registry](https://circleci.com/orbs/registry/) to see how the jobs are built.
+* In the examples on this page that use orbs, you will notice that the orbs are versioned with tags, for example, `aws-s3: circleci/aws-s3@x.y.z`. If you copy paste any examples you will need to edit `x.y.z` to specify a version. You can find the available versions listed on the individual orb pages in the [CircleCI Orbs Registry](https://circleci.com/orbs/registry/).
+* Any items in these examples that appear within `< >` should be replaced with your own parameters.
+
+## AWS
+
+This section covers deployment to S3, ECR/ECS (Elastic Container Registry/Elastic Container Service), as well as application deployment using AWS Code Deploy. For an in-depth look at deploying to AWS ECS from ECR, see the [Deploying to AWS ECS/ECR document]({{ site.baseurl }}/2.0/ecs-ecr/).
```

Comment:
All of the deploy examples look really good. I checked the ones I'm familiar with for accuracy. I don't know how anything about Azure, Capistrano, Cloud Foundry, or Firebase.

Reply:
Thanks so much for your help with this :-)